examples/chart/teleport-cluster/templates/pdb.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
```

Comment:
1.21 introduced `policy/v1` but I have kept v1beta1 so this chart can support earlier k8s versions.